Makefile

```diff
@@ -28,6 +28,9 @@ endif
 CGO_CFLAGS=-I/$(JAVA_HOME)/include -I/$(JAVA_HOME)/include/darwin
 GOBIN=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))build/bin
 GIT_COMMIT := $(shell git rev-parse --short HEAD)
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+GIT_LOCAL  := $(git rev-parse @)
```

Comment:
`shell` may be required before `git rev-parse @` and `git rev-parse origin`

Reply:
yeah